app/assets/javascripts/Remus.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function() {
+  $('[data-unit="create"]').on('submit', function(event) {
+    debugger
+    event.preventDefault();
+    $form = $(event.currentTarget);
+    $.ajax({
+      type: "POST",
+      url: $form.attr('action'),
+      // data: <SOMETHING>
+      dataType: 'json',
+      success: function(unit) {
+        action = '/units/'+ unit.id;
+        $newForm = $('<form>').attr({
+          action: action,
+          method: 'delete',
+          'data-unit': 'delete'
+        });
+        $unitButton = $('<input>').attr({type: 'submit', value: 'Delete Unit'});
+        $newForm.append($unitButton);
+        $unitListing = "<li class=\"unit\" data-unit-id=" +
+        unit.id + "> " + unit.name + "</li>"
+        $unitListing.append($newForm)
```

Comment:
Missing semicolon.<br>'$unitListing' is not defined.<br>'$newForm' is not defined.